src/js/helpers/handleScroll.js

```diff
@@ -78,9 +78,12 @@ const applyClasses = (el, top) => {
  * Scroll logic - add or remove 'aos-animate' class on scroll
  *
  * @param  {array} $elements         array of elements nodes
+ * @param  {node}  container         container
```

Comment:
The type is: `HTMLElement`.

See https://stackoverflow.com/a/21489264/4317384

Reply:
`container` could be a `Window` too, so maybe `{(HTMLElement|Window)}`? 😃

Reply:
Yes https://stackoverflow.com/a/35351628/4317384